gravitee-apim-console-webui/constants.json

```diff
@@ -1,3 +1,4 @@
 {
-  "baseURL": "/management/organizations/DEFAULT/environments/DEFAULT"
+  "baseURL": "/management/organizations/DEFAULT/environments/DEFAULT",
+  "trialBaseURL": "https://gravitee.io/self-hosted-trial"
```

Comment:
Do we really want to allow to change this?

Reply:
Ho you're right, I didn't see that file as opened to modification.

Reply:
@aelamrani I hardcoded the URL in the license service